example/ck_tile/03_gemm/run_gemm_example.inc

```diff
@@ -161,14 +161,82 @@ int run_gemm_example_with_layouts(int argc,
         c_m_n_gpu_ref.SetZero();
         c_m_n_gpu_buf_ref.SetZero();
 
+        ADataType* d_A;
+        BDataType* d_B;
+        CDataType* d_C;
+
+        hipError_t errA = hipMalloc(&d_A, M * K * sizeof(ADataType));
+        hipError_t errB = hipMalloc(&d_B, N * K * sizeof(BDataType));
+        hipError_t errC = hipMalloc(&d_C, M * N * sizeof(CDataType));
+        if(errA != hipSuccess)
```

Comment:
Can we create similar macro to include/ck/host_utility/hip_check_error.hpp:10 ?

Reply:
You're right, we have a function for that, fixed in latest change

Reply:
You forgot to add these change also in this file

Reply:
Good catch, thanks! Updated it also in that file